styles/style.css

```
@@ -0,0 +1,163 @@
+
+body {
+  background-color: whitesmoke;
+}
+
+h1.landing {
+  font-family: 'Saira Stencil One', cursive; 
+  font-size: 500%;
+  padding: 10%;
+  margin: 0 auto;
+}
+
+h2 {
+  font-size: xx-large;
+  font-family: 'Saira Stencil One', cursive; 
+}
+
+.center{
+  text-align: center;
+  padding-bottom: 5%;
+  padding-top: 5%
+}
+
+sub {
+  text-align: center;
+  font-size: xx-large;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+p.landing {
+  padding-top: 10%;
+}
+
+a {
+  padding: 15px;
+  color: darkslateblue;
+  text-decoration: none;
+  font-size: x-large;
+  border: 3px solid teal;
+  border-radius: 10px;
+  margin: 15%;
+}
+
+a:hover {
+  color: aquamarine;
+}
+
+li.nav {
+    list-style: none;
+    padding: 20px;
+    display: inline;
+    font-family: 'Saira Stencil One', cursive; 
+    font-size: x-large;
+
+}
+
+ul {
+  list-style: none;
+}
+ 
+ul.nav {
+  display: grid;
+  grid-template-columns: 4fr 1fr 1fr;
+}
+
+ul.footer {
+  margin: 0;
+  padding-left: 0;
+  
+}
+
+img.photos {
+  border-radius: 50%;
+  align-self: auto;
+}
+
+
+hr { 
+  display: block;
+  margin-top: 0.5em;
+  margin-bottom: 0.5em;
+  margin-left: auto;
+  margin-right: auto;
+  overflow: hidden;
+  border-style: inset;
+  border-width: 1px;
+}
+
+blockquote {
+  font-family: Georgia, serif;
+  font-size: 18px;
+  font-style: italic;
+  width: 800px;
+  line-height: 1.45;
+  position: center;
+}
+
+blockquote cite {
+  font-size: 14px;
+  display: block;
+  margin-top: 5px;
+}
+ 
+div.footer{
+padding: 10%
+
+}
+.container{ 
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr
+}
+
+.container li {
+  margin: 0;
+  padding: 0;
+}
+
```

Then adding a separate .container li and using grid. This is where you can design how the img relates to strong and relates to p. 
```suggestion
.container li {
display: grid;
grid-template-columns: (two columns at your choice of size);
grid-template-rows: (two rows at your choice of size);
}
```